include/dd/NoiseFunctionality.hpp

```diff
@@ -3,16 +3,18 @@
 * See file README.md or go to https://www.cda.cit.tum.de/research/quantum_dd/ for more information.
 */
 
-#ifndef DDSIM_NOISEFUNCTIONALITY_HPP
-#define DDSIM_NOISEFUNCTIONALITY_HPP
+#pragma once
 
 #include "dd/Definitions.hpp"
 #include "dd/Export.hpp"
+#include "operations/OpType.hpp"
+#include "operations/Operation.hpp"
 
 #include <random>
 #include <utility>
 
-using CN = dd::ComplexNumbers;
+using CN           = dd::ComplexNumbers;
+using arrayOfEdges = std::array<dd::DensityMatrixDD, std::tuple_size_v<decltype(dd::dNode::e)>>;
```

Comment:
Types should start with a capital letter.

Reply:
I changed that